src/Avalonia.Controls/Automation/Peers/PopupRootAutomationPeer.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Avalonia.Controls.Primitives;
+
+#nullable enable
+
+namespace Avalonia.Automation.Peers
+{
+    public class PopupRootAutomationPeer : WindowBaseAutomationPeer
+    {
+        public PopupRootAutomationPeer(PopupRoot owner)
```

Comment:
What is this AutomationPeer used for? Only for tracking the focus around popups?

Reply:
Yeah pretty much. It's almost the same as `WindowAutomationPeer` except that popups don't have a title and aren't control/content elements.